com.civicrm.hrjobroles/js/src/job-roles/controllers/hr-job-roles-controller.js

```diff
@@ -126,10 +126,10 @@ define([
              * @param end
              * @param {function} error
              */
-            $scope.validateDates = function (start, end, error) {
+            $scope.validateDates = function (start, end, contract_start, contract_end, error) {
```

Comment:
On a closer look I wonder also if it makes sense to have this function at all, you can incorporate its two-lines implementation in the `validateDates()` function you already wrote